Viper/Viper/CountriesScreen/View/CountriesViewController.swift

```diff
@@ -0,0 +1,82 @@
+// CountriesViewController.swift
+// Copyright © Koskin VA. All rights reserved.
+
+import UIKit
+
+/// Popular IT destination  screen
+final class CountriesViewController: UIViewController {
+    // MARK: - Visual Components
+
+    private var countriesCollectionView: UICollectionView {
+        let layout = UICollectionViewFlowLayout()
+        layout.sectionInset = UIEdgeInsets(top: 20, left: 10, bottom: 10, right: 10)
+        layout.itemSize = CGSize(width: view.frame.width, height: view.frame.width)
+        layout.scrollDirection = .vertical
+        let collectionView = UICollectionView(
+            frame: view.frame,
+            collectionViewLayout: layout
+        )
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(CountryCollectionViewCell.self, forCellWithReuseIdentifier: S.countryCollectionViewCell)
+        return collectionView
+    }
+
+    // MARK: - Private Properties
+
+    private let presenter: CountriesPresenterProtocol
+
+    // MARK: - Initializers
+
+    init(presenter: CountriesPresenterProtocol) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        title = S.countriesViewControllerTitle
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        view.addSubview(countriesCollectionView)
+    }
+}
+
+// MARK: - UICollectionViewDelegate, UICollectionViewDataSource
```

Comment:
```suggestion
// MARK: -  UICollectionViewDataSource
```
Лучше раздели делегат и датасорс, повышает читаемость